### DISCUSSION TITLE
datagrid筛选那期加载handycontrol报异常

### DISCUSSION THREAD

**Discussion by Monoideism** [BYJRK/DotNet-Discussions#31] [WPF 开发]
照着up  datagrid 数据筛选那期做，有筛选结果。
handycontrol 也能出效果
但是app.xaml里只要带上handycontrol 的Application.Resources那一段，就报异常：
System.IO.FileLoadException:“The given assembly name or codebase was invalid. (0x80131047)”

[datarid,ob.zip](https://github.com/BYJRK/DotNet-Discussions/files/10762656/datarid.ob.zip)

回去重敲一遍就没问题。。。

**Comment by notplay3** [comment]
可以试试重新生成
，或者把项目的 .vs文件删除后重新打开项目
